spec/unit/installer/xcode/single_pods_project_generator_spec.rb

```diff
@@ -441,117 +441,149 @@ class Xcode
               development_pod_targets = @generator.pod_targets.select { |pod_target| @generator.sandbox.local?(pod_target.pod_name) }
               @generator.share_development_pod_schemes(pod_generator_result.project, development_pod_targets)
             end
-          end
 
-          it 'shares test schemes' do
-            @generator.installation_options.
-                stubs(:share_schemes_for_development_pods).
-                returns(true)
-            @generator.sandbox.stubs(:development_pods).returns('CoconutLib' => fixture('CoconutLib'))
+            it 'shares test schemes' do
```

Comment:
same story with indentation and test here since its for single pods project generator.